<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janelaadicionarfluido</class>
 <widget class="QDialog" name="janelaadicionarfluido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbnTituloNome">
          <property name="text">
           <string>Nome:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbnTituloDensidade">
          <property name="text">
           <string>Densidade:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbnTituloViscosidade">
          <property name="text">
           <string>Viscosidade:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbnTituloProfundidadeInicial">
          <property name="text">
           <string>Profunidade Inicial:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbnTituloProfundidadeFinal">
          <property name="text">
           <string>Profundidade Final:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="LnValorNome"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LnValorDensidade"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LnValorViscosidade"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LnValorProfundidadeInicial"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LnValorProfundidadeFinal"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="btnReturn">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnReturn</sender>
   <signal>accepted()</signal>
   <receiver>janelaadicionarfluido</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReturn</sender>
   <signal>rejected()</signal>
   <receiver>janelaadicionarfluido</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
